ckanext/publicamundi/plugins.py

```diff
@@ -159,6 +159,10 @@ def before_map(self, mapper):
             m.connect(
                 '/api/publicamundi/dataset/export/{name_or_id}', 
                 action='dataset_export')
+
+            m.connect(
+                '/api/publicamundi/dataset/dcat/{name_or_id}',
```

Comment:
Well, i considered the original one as the default exporting action (as ISO is a meaningful name only for INSPIRE schema, not for other arbitrary schemata).